hydra-node/src/Hydra/Utils.hs

```diff
@@ -1,15 +1,27 @@
 module Hydra.Utils where
 
 import Crypto.Random (getRandomBytes)
-import Hydra.Cardano.Api (File (..), FileError, Key (SigningKey), getVerificationKey, writeFileTextEnvelope)
+import GHC.IO.Exception (userError)
+import Hydra.Cardano.Api (File (..), FileError (FileIOError), Key (SigningKey), getVerificationKey, writeFileTextEnvelope)
 import Hydra.Crypto (HydraKey, generateSigningKey)
 import Hydra.Options (GenerateKeyPair (..))
 import Hydra.Prelude
+import System.Directory (doesFileExist)
 import System.FilePath ((<.>))
 
 genHydraKeys :: GenerateKeyPair -> IO (Either (FileError ()) ())
 genHydraKeys GenerateKeyPair{outputFile} = do
-  sk :: SigningKey HydraKey <- generateSigningKey <$> getRandomBytes 16
-  runExceptT $ do
-    ExceptT $ writeFileTextEnvelope (File (outputFile <.> "sk")) Nothing sk
-    ExceptT $ writeFileTextEnvelope (File (outputFile <.> "vk")) Nothing (getVerificationKey sk)
+  fileExists <- doesFileExist outputFile
+  print fileExists
+  if fileExists
+    then
+      pure $
```

Comment:
If you are in `ExceptT` you could just `fail` or `throwM` perhaps?

Reply:
I wanted to keep the same return value since we handle it (actually just `die`) in the `Main`